Lib/inspect.py

```diff
@@ -662,8 +662,9 @@ def getfile(object):
         object = object.f_code
     if iscode(object):
         return object.co_filename
-    raise TypeError('{!r} is not a module, class, method, '
-                    'function, traceback, frame, or code object'.format(object))
+    raise TypeError('object of type {} is not a module, class, method, '
+                    'function, traceback, frame, or code object'.format(
+                        type(object).__name__))
```

Comment:
I am sorry I did not tell you in the previous message, but I realized just now.  You should align the `type(object).__name__` to `'function'`.